packages/entity-list/src/components/AdminSearchForm/QueryView.js

```diff
@@ -31,6 +39,11 @@ const QueryView = ({
   }
   const queryExists = !!query && query.trim().length > 0
   const queryHasErrors = queryError && Object.entries(queryError).length !== 0
+
+  const queryErrorValues = Object.keys(queryError || {}).map(key => queryError[key])
```

Comment:
Is there a reason to not just use Object.values?

Reply:
🤔  hmm good question.. not really. I will change it 👍